ppr-ui/src/components/parties/SecuredParties.vue

```diff
@@ -108,7 +108,7 @@
       <v-col>
         <v-data-table
           class="party-table"
-          :class="{ 'invalid-message': showErrorSecuredParties }"
+          :class="{ 'invalid-message': showErrorSecuredParties && !getSecuredPartyValidity() }"
```

Comment:
hey steph, I think this will only check ```!getSecuredPartyValidity()``` when it's first rendered since it's a fn. So it will calculate it when the page loads, but if the validity changes afterwards this won't update unless you change it to a computed or reactive value